src/goto-programs/string_abstraction.cpp

```diff
@@ -1046,13 +1046,13 @@ void string_abstractiont::move_lhs_arithmetic(exprt &lhs, exprt &rhs)
 
 goto_programt::targett string_abstractiont::abstract_pointer_assign(
   goto_programt &dest,
-  goto_programt::targett target)
+  const goto_programt::targett target)
```

Comment:
See other PR: I'm not sure this adds value.

Reply:
Well, these do add exactly as much value as on block-scoped variables, and we have a tendency to make these `const` unless they get mutated. There's of course more value on references.

Reply:
As a proposal, I'd say we add the `const` on the definition, but leave it out of the declaration (where it really doesn't do anything at all).